3.4/administration-cluster.md

```diff
@@ -208,3 +208,53 @@ To distribute shards onto the new _DBServer_ either click on the
 `Distribute Shards` button at the bottom of the `Shards` page in every
 database.
 
+This process can be monitored using the following javascript, that can give a 
```

Comment:
`This` refers to the cleanOutServer process, but the previous paragraph talks about rebalancing shards. Not sure if we should move the new content up above the paragraph `After the _DBServer_ has been cleaned out...`  or if it would suffice to change `This` to `The cleanOutServer process`.

Reply:
I guess that it makes sense to rename it as you said:

```
The cleanOutServer process
```

Will change it now